lib/components/Accordion/Item.js

```diff
@@ -92,35 +92,35 @@ Item.displayName = "Accordion.Item";
 
 Item.propTypes = {
   /**
-   * To specify a unique ID to the Accordion item.
+   * To specify a unique ID to the accordion item.
```

Comment:
@praveen-murali-ind IMO, we can keep component names' first letters capitalized. Lmk your thoughts.

Reply:
@deepakjosp _A Let’s use capital case for component names' first letter.

Reply:
@praveen-murali-ind  _a could you please share your thoughts on the above comment?